bundles/org.openhab.binding.tibber/README.md

```diff
@@ -77,7 +81,7 @@ Note: Tibber HomeId is retrieved from [developer.tibber.com](https://developer.t
 - If Tibber Pulse is connected, the Tibber API Explorer will report "true" for "realTimeConsumptionEnabled"
 - Copy HomeId from Tibber API Explorer, without quotation marks, and use this in the bindings configuration.
 
-```json
+```
```

Comment:
Please revert this change for proper syntax highlighting.

Reply:
Done. as this is not a valid json and IJ displayed a lot of errors, i've removed them.